packages/block-library/src/quote/edit.js

```diff
@@ -6,14 +6,17 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
 import {
 	AlignmentToolbar,
+	__experimentalBlock as Block,
 	BlockControls,
 	RichText,
 } from '@wordpress/block-editor';
-import { BlockQuotation } from '@wordpress/components';
 import { createBlock } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+import { BlockQuotation } from '@wordpress/primitives';
+
+const BlockWrapper = Block[ BlockQuotation ];
```

Comment:
This probably doesn't work on native-mobile. It's worth noting that `BlockQuotation` returns a string ("blockquote") on web and a `View` component on React Native. I have no idea how to get this working with a light block wrapper. Any ideas?